go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/cenkalti/backoff v2.0.0+incompatible
 	github.com/cpuguy83/go-md2man v1.0.8 // indirect
 	github.com/dustin/go-humanize v0.0.0-20180713052910-9f541cc9db5d
-	github.com/exoscale/egoscale v0.16.0
+	github.com/exoscale/egoscale v0.16.1-0.20190415122555-2143c08dc8f1
```

Comment:
Could you please release a tagged `v0.16.1` version? 😇

Reply:
Yep, I will do a release once https://github.com/exoscale/egoscale/pull/400 is merged (to also have this call in the release).

Reply:
done ;)